AdyenActions/AdyenActionComponent.swift

```diff
@@ -56,14 +56,6 @@ public final class AdyenActionComponent: ActionComponent, Localizable {
         }
     }
     
-    /// Dismiss any `DismissableComponent` managed by the `AdyenActionsComponent`, for example when payment has concluded.
-    ///
-    /// - Parameter animated: A boolean indicating whether to dismiss with animation or not.
-    /// - Parameter completion: A closure to execute when dismissal animation has completed.
-    internal func dismiss(_ animated: Bool, completion: (() -> Void)?) {
```

Comment:
This one I am still undeceive about:

in one hand - we never use it in our demo.
in the other - there is no harm in having it...

🤔

Reply:
it was requested specifically by a merchant :-)

Reply:
than I'll keep it :)